spec/models/comment_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe Comment, type: :model do
+  context "#have attributes" do
+    it "have attributes " do
+      expect(Comment.new.attributes.keys).to include("user_id", "content", "post_id")
```

Comment:
dùng symbol :user_id thay cho text "user_id" coi đươc em

Reply:
có lỗi ntn ạ:
Failure/Error: expect(Comment.new.attributes.keys).to include(:user_id, :content, :post_id)
     
       expected ["id", "content", "user_id", "post_id", "created_at", "updated_at"] to include :user_id, :content, and :post_id
       Diff:
       @@ -1,2 +1,2 @@
       -[:user_id, :content, :post_id]
       +["id", "content", "user_id", "post_id", "created_at", "updated_at"]